### DISCUSSION TITLE
Publishing

### DISCUSSION THREAD

**Discussion by ManeeshaChennoju** [community/community#62446] [New to GitHub]
Yeasterday i published my repository, and I unpublished again . and today I published again Im unable to see my published link in github pages. what might be the issue

**Comment by h7x4** [comment]
Hello @ManeeshaChennoju.

Just to make sure we're not talking about two different things, when you say that you *published* your repository, maybe you are referring to [making the repository public/private](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/setting-repository-visibility#changing-a-repositorys-visibility)? In that case, it will not have any effect on github pages.

Try following this guide, and come back to tell us if you're stuck: https://docs.github.com/en/pages/quickstart.

Also note that the last step might take up to 10 minutes.

**Comment by ManeeshaChennoju** [comment]
I published means, I mean that I deployed my project . In pages we can
found that published link of project right? so that is im asking im unable
to see that link. and thank you for reaching me out. My project was
published now. But can you tell me that how to update the projects code
after publishing. for example yesterday I deployed and again I unpublished
that , because im unable to update that. I hope you understand my problem
correctly.

Thankyou.

**Reply by h7x4** [reply]
Okay!

If the pages are configured in the standard way, whenever you change the code and push new commits GitHub pages should automatically update to reflect the new changes. Keep in mind that it might take a few minutes from when you push until the page is updated.

**Comment by ManeeshaChennoju** [comment]
Okay.....

Thank you for Clarification.
